src/background/registries/nearRegistry.ts

```diff
@@ -294,13 +296,28 @@ export class NearRegistry implements Registry {
     })
   }
 
+  public async getContextIds(moduleName: string): Promise<string[]> {
+    throw new Error('Method not implemented.')
+  }
+
   public async addContextId(moduleName: string, contextId: string) {
     await this._contract.addContextId({ contextId, moduleName })
   }
 
   public async removeContextId(moduleName: string, contextId: string) {
     await this._contract.removeContextId({ contextId, moduleName })
   }
+  public async getAdmins(moduleName: string): Promise<string[]> {
+    throw new Error('Method not implemented.')
+  }
+
+  public async addAdmin(moduleName: string, adressAdmin: string) {
+    await this._contract.addAdmin({ adressAdmin, moduleName })
+  }
+
+  public async removeAdmin(moduleName: string, adressAdmin: string) {
+    await this._contract.removeAdmin({ adressAdmin, moduleName })
+  }
```

Comment:
Remove it. The NEAR smart-contract of dapplet registry doesn't contain admin functionality.